src/dex-ui/store/poolsSlice/poolsSlice.ts

```diff
@@ -211,6 +217,41 @@ const createPoolsSlice: PoolsSlice = (set, get): PoolsStore => {
       }
       app.setFeaturesAsLoaded(["addLiquidityTransactionState"]);
     },
+    fetchCreatePoolTransactionFee: async () => {
```

Comment:
Please move this to a React Query hook. We need to move away from using Zustand for managing server state.

Reply:
Done